x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/feature_table.tsx

```diff
@@ -36,6 +38,10 @@ import type { PrivilegeFormCalculator } from '../privilege_form_calculator';
 import { ChangeAllPrivilegesControl } from './change_all_privileges';
 import { FeatureTableExpandedRow } from './feature_table_expanded_row';
 
+const subFeatureTabSpacing: CSSObject = {
+  'margin-left': '30px',
+};
```

Comment:
I believe this needs to be `28px` in order to align to the left edge of the text. Additionally, you can also import the `useEuiTheme` hook (from `@elastic/eui`) and `css` (from `@emotion/react`) to do the following. Note that if you decide to do so, you'll probably need to move this to `renderPrivilegeControlsForFeature` function component, as I don't think the hook can be used at the top level.

```suggestion
const { euiTheme } = useEuiTheme();
const subFeatureTabSpacing = css`
  margin-left: ${euiTheme.size.l} + ${euiTheme.size.xs};
`;
```

Reply:
gotcha thanks for the suggestion!